docs/docs/deploy/aws-app-runner.md

```diff
@@ -0,0 +1,92 @@
+---
+sidebar_position: 2
+---
+
+# AWS App Runner 🏃
+
+[App Runner](https://aws.amazon.com/apprunner/) is a fully managed service that makes it easy for developers to quickly deploy containerized web applications and APIs, at scale and with no prior infrastructure experience required. It is service in [Amazon Web Services](https://aws.amazon.com/). App Runner automatically:
+
+- Load balances traffic with encryption
+- Scales to meet your traffic needs
+- Makes it easy for your services to communicate with other AWS services and applications that run in a private Amazon VPC
+
+## Prerequisites
+
+Before you get started, if you haven't already completed these steps, you'll have to:
+
+1. Create a free [Amazon Web Services account](https://docs.aws.amazon.com/accounts/latest/reference/manage-acct-creating.html)
+
+:::caution
+While Amazon Web Services has a free tier that should cover testing projects, you can incur costs when running this quickstart through App Runner or Elastic Container Registry. For more details, see [AWS App Runner Pricing](https://aws.amazon.com/apprunner/pricing/) and [Amazon Elastic Container Registry Pricing](https://aws.amazon.com/ecr/pricing/).
+:::
+
+2. Install [Docker](https://docs.docker.com/get-docker/) on your computer, and you'll have to start app. You can verify it is set up correctly by running:
+
+```bash
+docker images
+```
+
+(If Docker is running, the command will print the images on your computer. If not, it will print something like `Cannot connect to the Docker daemon at unix:///var/run/docker.sock. Is the docker daemon running?`)
```

Comment:
```suggestion
(If Docker is running, the command will print the images on your machine. If not, it will print something like `Cannot connect to the Docker daemon at unix:///var/run/docker.sock. Is the docker daemon running?`)
```